### PR TITLE
Component/ in radius

### PR DESCRIPTION
**Pull request by julialoeschel**
- create input for Radius
- store value in meters for further use
- log the meters 

Bildschirmfoto 2021-12-09 um 15 22 38

### CODE REVIEW

File: src/app/pages/Details/Details.tsx [hunk 1]
```diff
@@ -13,6 +14,10 @@ export default function Details(): JSX.Element {
 
   localStorage.setItem('CominFromDetailsPage', 'true')
 
+  function getRadius(x: number) {
+    console.log('Search Radius in meters', x)
```
Comment (janhendrikstruve) [inline src/app/pages/Details/Details.tsx:18]: Console.log 🔍 still needed?

Comment (janhendrikstruve) [inline src/app/pages/Details/Details.tsx:18]: Ich glaube "radius" gehört klein, bin mir aber nicht sicher 🦉

File: src/app/components/InRadius/InRadius.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const [radius, setRadius] = useState(0)
+
+  getRadius(radius)
+
+  return (
+    <>
+      <RadiusText>
+        Search Radius{' '}
+        <RadiusInput
+          type="number"
```
Comment (marwinburesch) [inline src/app/components/InRadius/InRadius.tsx:18]: You could use [range](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) here :)

**Review by janhendrikstruve — APPROVED** [review]
Great code! Just two things i'm not sure about 🦆

File: src/app/components/InRadius/InRadius.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import styled from 'styled-components'
+
+type InRadiusProps = {
+  getRadius: (radius: number) => void
+}
+
+export default function InRadius({ getRadius }: InRadiusProps): JSX.Element {
+  const [radius, setRadius] = useState(0)
+
+  getRadius(radius)
```
Comment (marwinburesch) [inline src/app/components/InRadius/InRadius.tsx:11]: In the future, it's probably best to wrap this in a condition and/or maybe use `useEddect`?

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/app/pages/Details/Details.tsx (hunk 1 above)
Comment (julialoeschel) [reply src/app/pages/Details/Details.tsx:18]: for now: Yes, right now its the only thing the fuction does :)

File: src/app/components/InRadius/InRadius.tsx (hunk 2 above)
Comment (julialoeschel) [reply src/app/components/InRadius/InRadius.tsx:18]: great, thats what i needed, and its easyer than I thought!

**Pull request merged**